tests/model_serving/model_runtime/vllm/utils.py

```diff
@@ -132,3 +142,49 @@ def validate_supported_quantization_schema(q_type: str) -> None:
 def validate_inference_output(*args: tuple[str, ...], response_snapshot: Any) -> None:
     for data in args:
         assert data == response_snapshot, f"output mismatch for {data}"
+
+
+def validate_raw_openai_inference_request(
+    pod_name: str,
+    isvc: InferenceService,
+    response_snapshot: Any,
+    chat_query: list[list[dict[str, str]]] = CHAT_QUERY,
+    completion_query: list[dict[str, str]] = COMPLETION_QUERY,
```

Comment:
using mutable default values ​​as default arguments is not recommended.

Reply:
I have  have taken suggestion  into consideration. However, the suggestion doesn't make a difference in this case. It’s not recommended when we play around with mutable object and modify it  but in our code, we’re simply passing the value around without performing any  changes/operation. I understand the ambiguity around using a default parameter with a mutable object, but technically, that doesn’t apply here.